### DISCUSSION TITLE
Doc/README: should all Templates include Inspect?

### DISCUSSION THREAD

**Discussion by tomByrer** [statelyai/xstate#1888] [Ideas]
I would say Inspect is good enough now for many use-cases that will end up on CodeSandbox.  Should all the Templates in the [README](https://github.com/davidkpiano/xstate#templates) & [docs](https://xstate.js.org/docs/#templates) have their links switched over the the Inspect versions?

BTW, *I personally* find CodeSandbox & esp the Inspect there too frustrating to use; kinda triggers my concussion symptom a bit because how flashy the screen updates are (pehaps dark mode?)..  But I'm sure for 99.5% of the world, they are great.

**Comment by davidkpiano** [comment]
I think it would be useful to have a `(+ inspect)` link on the side of the normal template links. Best of both worlds!

**Reply by tomByrer** [reply]
Might be [too many choices](https://youtu.be/VO6XEQIsCoM?list=PLakykuPxo3cgxD8wBhPuKop594pq0C9pp&t=25).

**Reply by davidkpiano** [reply]
> Might be [too many choices](https://youtu.be/VO6XEQIsCoM?list=PLakykuPxo3cgxD8wBhPuKop594pq0C9pp&t=25).

This is not a case of "which choice should I make", it's a case of "I know exactly what I want (some framework + visualization), where can I find it", which is why the "too many choices" issue doesn't apply here.
